heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraContext.java

```diff
@@ -14,16 +14,26 @@
 
 package com.twitter.heron.scheduler.aurora;
 
+import java.io.File;
+
 import com.twitter.heron.spi.common.Config;
 import com.twitter.heron.spi.common.Context;
 
 public final class AuroraContext extends Context {
   public static final String JOB_LINK_TEMPLATE = "heron.scheduler.job.link.template";
+  public static final String JOB_TEMPLATE = "heron.scheduler.job.template";
+  public static final String JOB_TEMPLATE_DIR = "heron.scheduler.job.template.dir";
 
   private AuroraContext() {
   }
 
   public static String getJobLinkTemplate(Config config) {
     return config.getStringValue(JOB_LINK_TEMPLATE);
   }
+
+  public static String getHeronAuroraPath(Config config) {
+    return new File(config.getStringValue(JOB_TEMPLATE_DIR, Context.heronConf(config)),
+        config.getStringValue(JOB_TEMPLATE,"heron.aurora")).getPath();
+  }
+
```

Comment:
@chris-pardy - in the case of both the config being absent, how does this work? Is the JOB_TEMPLATE_DIR set to default somewhere?